apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -120,9 +120,35 @@
     </div>
 
     <div *ngIf="displayApi$ | async">
-      <datahub-record-apis class="block mt-6"></datahub-record-apis>
+      <datahub-record-apis
+        (selectedLink)="selectCard($event)"
+        class="block mt-6"
+      ></datahub-record-apis>
     </div>
   </div>
+  <div
+    class="bg-primary-opacity-10 pt-8 overflow-visible sm:pt-5"
+    *ngIf="selectedCard$ | async"
+  >
+    <div class="flex flex-col container-lg px-4 lg:mx-auto gap-6">
+      <div class="flex flex-wrap justify-between mb-6 sm:mb-2 ng-star-inserted">
+        <p class="text-[28px] text-title font-title" translate>
+          Générer une URL personnalisée
```

Comment:
You should take the habit to not write french text in the template, it's a source for missing some translation :upside_down_face:

Reply:
Translations are not done yet :slightly_smiling_face: